solr/solr-ref-guide/src/schema-api.adoc

```diff
@@ -844,12 +827,27 @@ If `multiValued` is defined as either true or false (most likely true), that wil
 
 ==== List Fields Examples
 
-Get a list of all fields.
+[.dynamic-tabs]
+--
+[example.tab-pane#v1list-fields]
+====
+[tab-label]*V1 API*
+[source,bash]
+----
+http://localhost:8983/techproducts/schema/fields
+----
+====
 
+[example.tab-pane#v2list-fields]
+====
+[tab-label]*V2 API*
 [source,bash]
 ----
-curl http://localhost:8983/solr/gettingstarted/schema/fields
+
+curl -X GET "http://localhost:8983/api/collections/techproducts/schema/fields?wt=json"
```

Comment:
[Q] Is the `wt=json` here redundant, or does the schema API really return something other than JSON by default?

Reply:
I don't know why I thought I was getting xml!   I really appreciate the eagle eyes...